src/pages/publication/references_tab/JournalArticleForm.tsx

```diff
@@ -13,11 +13,19 @@ import DoiField from './components/DoiField';
 import SelectTypeField from './components/SelectTypeField';
 import PublisherField from './components/PublisherField';
 
+const StyledContent = styled.div`
+  display: grid;
+  gap: 1rem;
+`;
+
 const StyledArticleDetail = styled.div`
   display: grid;
-  grid-template-columns: auto auto auto auto auto auto;
+  grid-template-areas: 'volume issue from to or article';
   grid-column-gap: 1rem;
   align-content: center;
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    grid-template-areas: 'volume' 'issue' 'from' 'to' 'or' 'article';
```

Comment:
trengs dette? siden den er lik det på linj 23

Reply:
ikke lik. notice the `''`

Reply:
åja stemmer, det betyr ny linje det?

Reply:
jepp!